jsapp/js/components/formLanding.es6

```diff
@@ -129,8 +134,31 @@ export class FormLanding extends React.Component {
       asset: this.state
     });
   }
+  increaseDVCOUNT_LIMIT() {
+    if (this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM <= this.state.deployed_versions.count + DVCOUNT_LIMIT_MINIMUM) {
+      this.setState({ DVCOUNT_LIMIT : this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM });
+    }
+    if (this.state.DVCOUNT_LIMIT >= 80) {
```

Comment:
Leftover from the first time I tried my solution, I needed to call `dataInterface` once the next time a user clicks it would need to load the next 100. Removed now